<ui version="4.0" >
 <class>KopeteIdentityConfigBase</class>
 <widget class="QWidget" name="KopeteIdentityConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manage Identities</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="mIdentityList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Identity ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="mButtonNew" >
       <property name="toolTip" >
        <string>Add new account</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mButtonRemove" >
       <property name="toolTip" >
        <string>Remove selected account</string>
       </property>
       <property name="whatsThis" >
        <string>Remove selected account</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mButtonDefault" >
       <property name="text" >
        <string>Set as &amp;Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mButtonEdit" >
       <property name="toolTip" >
        <string>Modify selected account</string>
       </property>
       <property name="whatsThis" >
        <string>Let you edit the account's properties.</string>
       </property>
       <property name="text" >
        <string>&amp;Modify...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
